### PR TITLE
Sidebar2

### PR DESCRIPTION
**Pull request by mlorhan**
Cleaning up the sidebar, getting rid of unused buttons, adding a tools section which currently just has Analyst Calls, truncating username if it's too long, setting a website name at the top left, replacing default user icon with a circle around the first letter of the username.

### CODE REVIEW

File: back-end/services/user.py [hunk 1]
```diff
@@ -1 +1,68 @@
-# skipping this one since the return is fairly complex and I don't want to mess anyone elses code up
\ No newline at end of file
+# skipping this one since the return is fairly complex and I don't want to mess anyone elses code up
```
Comment (schuywalker) [inline back-end/services/user.py:1]: can delete this first comment line but not a big deal

File: back-end/services/watchlist.py [hunk 2]
```diff
@@ -35,34 +38,44 @@ def createWatchlists(userID, watchlistName:str):
...
+            if (len(tickers) > 0):
+                WatchlistService.addTickersToWatchlist(wl_id, user_ID, True, *tickers)
+                ret = wl_id
+
         except Error as e:
             if (e.errno == errorcode.ER_DUP_ENTRY):
                 ret = ("Error: Duplicate Entry",409)
             else:
-                ret = ("Error: ",500)
+                ret = ("Error: ",e,500)
```
Comment (schuywalker) [inline back-end/services/watchlist.py:57]: this is my bad - should be 400. I think you can commit the change in my comment below if you approve my review?

Comment (schuywalker) [inline back-end/services/watchlist.py:57]:
```suggestion
                ret = ("Error: ",e,400)
```

**Review by schuywalker — APPROVED** [review]
This Pull Request includes the bug fixes in watchlist.py endpoints, changes to fit db schema, also all the account stuff.

**Comment by mszym99** [comment]
Looks good.

**Pull request merged**